internal/pkg/agent/application/upgrade/artifact/download/http/verifier.go

```diff
@@ -115,13 +116,13 @@ func (v *Verifier) Verify(a artifact.Artifact, version string, skipDefaultPgp bo
 	return nil
 }
 
-func (v *Verifier) verifyAsc(a artifact.Artifact, version string, skipDefaultKey bool, pgpSources ...string) error {
-	filename, err := artifact.GetArtifactName(a, version, v.config.OS(), v.config.Arch())
+func (v *Verifier) verifyAsc(a artifact.Artifact, aVersion version.ParsedSemVer, skipDefaultKey bool, pgpSources ...string) error {
+	filename, err := artifact.GetArtifactName(a, aVersion, v.config.OS(), v.config.Arch())
```

Comment:
nit: all these renames from `version` to `aVersion` are making it harder to pick out the meaningful changes.

Reply:
Well there was a name conflict with the `version` package which I tried to disambiguate. Would you prefer a rename of the package import perhaps ?

Reply:
Renamed the imports and restored `version` variable names